bitnami/zookeeper/templates/statefulset.yaml

```diff
@@ -25,6 +25,9 @@ spec:
       name: {{ template "zookeeper.fullname" . }}
       labels: {{- include "zookeeper.labels" . | nindent 8 }}
         app.kubernetes.io/component: zookeeper
+        {{- if .Values.podLabels }}
+{{ toYaml .Values.podLabels | indent 8 }}
```

Comment:
Please change this to use the following function from the helpers:
```
{{/*
Renders a value that contains template.
Usage:
{{ include "zookeeper.tplValue" ( dict "value" .Values.path.to.the.Value "context" $) }}
*/}}
```

Reply:
Fixed.  Would you like me to add the `(evaluated as a template)` in the README to templated sections, similar to the Kafka chart?